Tasks/DockerV2/Tests/L0.ts

```diff
@@ -179,14 +179,14 @@ describe("DockerV2 Suite", function () {
         process.env[shared.TestEnvVars.containerRegistry] = "dockerhubendpoint";
         process.env[shared.TestEnvVars.repository] = "testuser/testrepo";
         process.env[shared.TestEnvVars.command] = shared.CommandTypes.build;        
-        process.env[shared.TestEnvVars.tags] = "tag1,tag2\ntag3\ntag4";
+        process.env[shared.TestEnvVars.tags] = "tag1,tag2\ntag3";
```

Comment:
Add one for the earlier case you mentioned as well. ie.`\n\n`

Reply:
Done